src/content/en/tools/chrome-devtools/network-performance/network-conditions.md

```diff
@@ -2,24 +2,32 @@ project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
 description: It’s easy to overlook the network conditions your users will face on mobile. Use DevTools to emulate different network conditions. Fix any load time issues and your users will thank you.
 
-{# wf_updated_on: 2015-07-20 #}
+{# wf_updated_on: 2016-12-28 #}
 {# wf_published_on: 2015-04-13 #}
 
 # Optimize Performance Under Varying Network Conditions {: .page-title }
```

Comment:
I didn't want to redirect users to one of the new docs, because it's not a 1:1 migration. Instead, I put a link at the top of each section indicating where to find related info in the new docs. I think this is a much more responsible migration approach and I'm very excited about it. Also note that this doc is no longer in the ToC.